### PR TITLE
feat: disable amplitude and sentry when using fedramp endpoints

### PR DESCRIPTION
**Pull request by j-luong**
### Description

_This PR should disable analytics and sentry events when using a fedramp endpoint._

### Checklist

- [x] Tests added and all succeed
- [x] Linted
- [x] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### CODE REVIEW

File: src/test/integration/analytics.test.ts [hunk 1]
```diff
@@ -34,4 +34,12 @@ suite('Analytics', () => {
 
     strictEqual(welcomeIsViewed.notCalled, true);
   });
+
+  test('"Welcome Is Viewed" not tracked if using fedramp endpoint', async () => {
+    await configuration.setEndpoint('https://snyk.fedramp.gov');
```
Comment (Avishagp) [inline src/test/integration/analytics.test.ts:39]: I'm getting "Unsafe call of an `any` typed value" trying to run this test 🤔

Comment (bastiandoetsch) [reply src/test/integration/analytics.test.ts:39]: Huh. Just adjusted one of the above. And works on CI and my computer.

Comment (bastiandoetsch) [reply src/test/integration/analytics.test.ts:39]: On which line does your problem occur?

Comment (j-luong) [reply src/test/integration/analytics.test.ts:39]: I've left the test in so when we tackle HEAD-700, this feature will be tested

**Review by Avishagp — APPROVED** [review]

**Pull request merged**